Documentation/contributing/development/dev_setup.rst

```diff
@@ -317,6 +317,22 @@ to enable NFS.
       # vers3=y
       ...
 
+.. note::
+
+   A later version of VirtualBox (v6.1.28 or above) restricts IP ranges
+   for host-only networks by default. Make sure IP ranges used for VMs
+   are specified explicitly:
+
+   .. code-block:: none
+
+      # cat /etc/vbox/networks.conf
+      ...
+      * 192.168.56.0/21
+      * FD00::/16
+      ...
+
+   See `VirtualBox Host-Only Networking <https://www.virtualbox.org/manual/ch06.html#network_hostonly>`_ for more detail.
```

Comment:
How did you hit this issue and what was the exact error? This should have been fixed in https://github.com/cilium/cilium/pull/17900 by using different IP address ranges.

Reply:
Yes, let's fix the script instead.

@nbusseneau I assume we'll also need a fix for our CI infrastructure since the fix isn't in v1.9?

Reply:
Yes, we need to fix both the dev (top-level) and CI (under `/test`) `Vagrantfile` to ensure the IPv6 ranges are sound (if they are not already). I recall we discussed that at some point while fixing IPv4, but I never got around to check. Thank you very much for looking into this @koncha99 :)